v2/components/DashboardCalloutCard.tsx

```diff
@@ -2,11 +2,26 @@ import { FC } from "react";
 import Link from "next/link";
 import { useTranslation, Trans } from "next-i18next";
 import { ArrowRightIcon } from "@heroicons/react/outline";
-
 import { formatPercentage } from "../utils";
+import APYTooltip from "./APYTooltip";
+import { CoreSelectors } from "v2/store/core";
+import { useSelector } from "react-redux";
+import { JarDetails } from "v2/types";
+import { ChainNetwork } from "picklefinance-core";
 
 const DashboardCalloutCard: FC = () => {
   const { t } = useTranslation("common");
+  let coreMaxApy: JarDetails | undefined = useSelector(
+    CoreSelectors.selectMaxApy,
+  );
+
+  if (!coreMaxApy)
+    coreMaxApy = {
+      name: "",
+      apiKey: "",
+      chain: "Ethereum",
```

Comment:
Can we fetch this constant from pf-core? (I'm not sure but would be nice)

Reply:
The Chains enum from pf-core only provides "eth", and not the preferred display name of "Ethereum" - getting this would be more trouble than it's worth imo.